charts/nd-common/README.md

```diff
@@ -2,7 +2,7 @@
 
 A helper chart used by most of our other charts
 
-![Version: 0.0.6](https://img.shields.io/badge/Version-0.0.6-informational?style=flat-square) ![Type: library](https://img.shields.io/badge/Type-library-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
```

Comment:
@saansh45 Can you add a docstring into `README.md.gotpl` that desribes the new `containerName` function

Reply:
done